alacritty/src/logging.rs

```diff
@@ -35,10 +34,13 @@ pub const LOG_TARGET_CONFIG: &str = "alacritty_config_derive";
 const ALACRITTY_EXTRA_LOG_TARGETS_ENV: &str = "ALACRITTY_EXTRA_LOG_TARGETS";
 
 /// User configurable extra log targets to include.
-static EXTRA_LOG_TARGETS: Lazy<Vec<String>> = Lazy::new(|| {
-    env::var(ALACRITTY_EXTRA_LOG_TARGETS_ENV)
-        .map_or(Vec::new(), |targets| targets.split(';').map(ToString::to_string).collect())
-});
+static EXTRA_LOG_TARGETS: OnceLock<Vec<String>> = OnceLock::new();
```

Comment:
Guess you can move this into the function itself?

Reply:
Yes, TIL